BDNS_Abbreviations_Register.csv

```diff
@@ -29,6 +29,7 @@ av equipment - iptv content management system device,CMS,IfcAudioVisualAppliance
 av equipment - networked video recorder,NVR,IfcAudioVisualAppliance,PLAYER
 av equipment - speaker,SPK,IfcAudioVisualAppliance,SPEAKER
 av equipment - video wall,VDW,IfcAudioVisualAppliance,DISPLAY
+battery,BAT,IfcElectricFlowStorageDevice,BATTERY
```

Comment:
Please change the abbreviation to BATT so that it also matches the [related Digital Buildings Ontology entity type](https://github.com/google/digitalbuildings/blob/d98aa29530ec80375247722d26824b97f326a44f/ontology/yaml/resources/ELECTRICAL/entity_types/GENERALTYPES.yaml#L15).

Reply:
Updated